servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -0,0 +1,141 @@
+    }
+
+    /**
+     * A builder for creating instances of {@link DefaultHttpLoadBalancerFactory}.
+     *
+     * @param <ResolvedAddress> The type of address after resolution for the {@link HttpLoadBalancerFactory} built by
+     * this builder.
+     */
+    public static final class Builder<ResolvedAddress> {
+        private final LoadBalancerFactory<ResolvedAddress, FilterableStreamingHttpLoadBalancedConnection> rawFactory;
+        private final HttpExecutionStrategyInfluencer strategyInfluencer;
+
+        private Builder(
+                final LoadBalancerFactory<ResolvedAddress, FilterableStreamingHttpLoadBalancedConnection> rawFactory,
+                final HttpExecutionStrategyInfluencer strategyInfluencer) {
+            this.rawFactory = rawFactory;
+            this.strategyInfluencer = strategyInfluencer;
+        }
+
+        /**
+         * Builds a {@link DefaultHttpLoadBalancerFactory} using the properties configured on this builder.
+         *
+         * @return A {@link DefaultHttpLoadBalancerFactory}.
+         */
+        public DefaultHttpLoadBalancerFactory<ResolvedAddress> build() {
+            return new DefaultHttpLoadBalancerFactory<>(new Config<>(rawFactory, strategyInfluencer));
+        }
+
+        /**
+         * Creates a new {@link Builder} instance using the default {@link LoadBalancer} implementation.
+         *
+         * @param <ResolvedAddress> The type of address after resolution for the {@link LoadBalancerFactory} built by
+         * the returned builder.
+         * @return A new {@link Builder}.
+         */
+        public static <ResolvedAddress> Builder<ResolvedAddress> fromDefaults() {
+            return from(newRoundRobinFactory());
+        }
+
+        /**
+         * Creates a new {@link Builder} using the passed {@link LoadBalancerFactory}.
+         *
+         * @param rawFactory {@link LoadBalancerFactory} to use for creating a {@link HttpLoadBalancerFactory} from the
+         * returned {@link Builder}.
+         * @param <ResolvedAddress> The type of address after resolution for a {@link HttpLoadBalancerFactory} created
+         * by the returned {@link Builder}.
+         * @return A new {@link Builder}.
+         */
+        public static <ResolvedAddress> Builder<ResolvedAddress> from(
+                final LoadBalancerFactory<ResolvedAddress, FilterableStreamingHttpLoadBalancedConnection> rawFactory) {
+            final HttpExecutionStrategyInfluencer strategyInfluencer;
+            if (rawFactory instanceof HttpExecutionStrategyInfluencer) {
+                strategyInfluencer = (HttpExecutionStrategyInfluencer) rawFactory;
+            } else if(rawFactory instanceof RoundRobinLoadBalancerFactory) {
+                strategyInfluencer = strategy -> strategy; // RoundRobinLoadBalancer is non-blocking.
+            } else {
+                /* user provided load balancer assumed to be blocking unless told otherwise */
```

Comment:
nit: consider using `//` comments

Reply:
nit: consider clarifying, the only supported way to override the `rawFactory` to implement `HttpExecutionStrategyInfluencer`.